src/util.ts

```diff
@@ -159,38 +164,35 @@ function getManifestsFromSideloadingDirectory(inputApplication: string): Promise
   });
 }
 
-function removeManifestFromSideloadingDirectory(inputApplication: string, manifestPathToRemove: string): Promise<any> {
-  return new Promise(async (resolve, reject) => {
-    try {
-      let manifestRemoved = false;
+async function removeManifestFromSideloadingDirectory(inputApplication: string, manifestPathToRemove: string): Promise<void> {
+  try {
+    let manifestRemoved = false;
+    for (let application of Object.keys(applicationProperties)) {
+      if (!inputApplication || application === inputApplication) {
+        const sideloadingDirectory = applicationProperties[application].sideloadingDirectory;
 
-      for (let application of Object.keys(applicationProperties)) {
-        if (!inputApplication || application === inputApplication) {
-          const sideloadingDirectory = applicationProperties[application].sideloadingDirectory;
+        if (!fs.existsSync(sideloadingDirectory)) {
+          continue;
+        }
 
-          if (!fs.existsSync(sideloadingDirectory)) {
-            continue;
+        const sideloadingManifestPath = await getSideloadManifestPath(manifestPathToRemove, application);
+        fs.readdirSync(sideloadingDirectory).forEach(manifestName => {
+          const realManifestPath = (fs.realpathSync(path.join(sideloadingDirectory, manifestName)));
+          if (sideloadingManifestPath === realManifestPath) {
+            console.log(`Removing ${sideloadingManifestPath} for application ${application}`);
+            fs.unlinkSync(sideloadingManifestPath);
+            manifestRemoved = true;
           }
-
-          fs.readdirSync(sideloadingDirectory).forEach(async manifestName => {
-            const realManifestPath = (fs.realpathSync(path.join(sideloadingDirectory, manifestName)));
-            const sideloadedManifestPath = await getSideloadManifestPath(manifestPathToRemove, application);
-            if (sideloadedManifestPath === realManifestPath) {
-              console.log(`Removing ${sideloadedManifestPath} for application ${application}`);
-              fs.unlinkSync(sideloadedManifestPath);
-              manifestRemoved = true;
-            }
-          });
-        }
+        });
       }
-
-      if (!manifestRemoved) {
-        return reject('No manifests were found to remove. Use "list" to show manifests that have been added.');
+      if (manifestRemoved) {
+        return;
       }
-    } catch (err) {
-      reject(err);
     }
-  });
+    return console.log('No manifests were found to remove. Use "list" to show manifests that have been added.');
+  } catch (err) {
+    throw new Error(`removeManifestFromSideloadingDirectory failed with error:\n${err}`);
```

Comment:
Suggest: `Unable to remove the manifest from the sideloading directory.\n${err}`